src/App.css

```diff
@@ -16,17 +16,15 @@
 .canvas {
   background-color: var(--element-background);
   position: relative;
-  height: 60vmin;
-  width: 60vmin;
-  min-height: 60vmin;
-  min-width: 60vmin;
+  height: 70vmin;
+  width: 70vmin;
```

Comment:
70 does not fit the screen along with graph-editor without scrolling

Reply:
Imo it would be better if canvas and graph editor was in a row on desktop.

Reply:
I'll go back to 60 then, life is too short to mess with CSS